internal/provwasm/stargate_whitelist.go

```diff
@@ -48,44 +59,132 @@ func init() {
 	// ==========================================================
 
 	// auth
+	setWhitelistedQuery("/cosmos.auth.v1beta1.Query/Accounts", &authtypes.QueryAccountsResponse{})
 	setWhitelistedQuery("/cosmos.auth.v1beta1.Query/Account", &authtypes.QueryAccountResponse{})
+	setWhitelistedQuery("/cosmos.auth.v1beta1.Query/AccountAddressByID", &authtypes.QueryAccountAddressByIDResponse{})
 	setWhitelistedQuery("/cosmos.auth.v1beta1.Query/Params", &authtypes.QueryParamsResponse{})
+	setWhitelistedQuery("/cosmos.auth.v1beta1.Query/ModuleAccounts", &authtypes.QueryModuleAccountsResponse{})
+	setWhitelistedQuery("/cosmos.auth.v1beta1.Query/ModuleAccountByName", &authtypes.QueryModuleAccountByNameResponse{})
+	setWhitelistedQuery("/cosmos.auth.v1beta1.Query/Bech32Prefix", &authtypes.Bech32PrefixResponse{})
+	setWhitelistedQuery("/cosmos.auth.v1beta1.Query/AddressBytesToString", &authtypes.AddressBytesToStringResponse{})
+	setWhitelistedQuery("/cosmos.auth.v1beta1.Query/AddressStringToBytes", &authtypes.AddressStringToBytesResponse{})
+	setWhitelistedQuery("/cosmos.auth.v1beta1.Query/AccountInfo", &authtypes.QueryAccountInfoResponse{})
+
+	// authz
+	setWhitelistedQuery("/cosmos.authz.v1beta1.Query/Grants", &authztypes.QueryGrantsResponse{})
+	setWhitelistedQuery("/cosmos.authz.v1beta1.Query/GranterGrants", &authztypes.QueryGranterGrantsResponse{})
+	setWhitelistedQuery("/cosmos.authz.v1beta1.Query/GranteeGrants", &authztypes.QueryGranteeGrantsResponse{})
```

Comment:
This is an incorrect assessment.  The reason these queries have been added is explicitly because they are allowed to be used.  If they were not listed they would not be allowed per the security [deterministic execution] requirements.